yelp.py

```diff
@@ -0,0 +1,32 @@
+import requests
+import json
+import logging
+
+api_key='<CHECK SLACK CHAT FOR THIS>'
+url = 'https://api.yelp.com/v3/businesses/search'
+
+
+# Makes request to the yelp API using the search_term as a parameter
+# returns the response in json format, to be used in the "get_business_name(data)" function
+def api_request(search_term):
+    try:
+        headers = {'Authorization':'Bearer %s' % api_key}
+        params = {'term':'%s' % search_term,'location':'Minneapolis'}
+        req = requests.get(url, params=params, headers=headers)
+        data = req.json()
+        return data
+    # Researched Try/Except functions for requests, information found at URL below:
+    # https://stackoverflow.com/questions/16511337/correct-way-to-try-except-using-python-requests-module
+    except requests.exceptions.Timeout:
+        log('Request Timed Out: ' + req)
+    except requests.exceptions.TooManyRedirects:
+        log('Redirect error- URL may be bad. Check URL, headers, params')
+    except requests.exceptions.RequestException as e:
+        log('Catastrophic error - exiting program')
+        raise SystemExit(e)
```

Comment:
The SystemExit might completely shutdown the app when it's integrated into flask. We can cross that bridge when we get there.

You _might_ need a catchall exception at the end in case there's an exception thrown other than those three.

Reply:
@whagestad I'll approve and merge in the interest of moving things along